meetings/2022-11-07.md

```diff
@@ -0,0 +1,107 @@
+* Repository: https://github.com/solid/authentication-panel
+
+## Present
+* Aaron Coburn
+* elf Pavlik
+* Emelia Smith
+* Wouter Termont
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Use panel chat and repository. Queue in call to talk.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md)
+* If this is your first time, welcome! please introduce yourself.
+
+### Scribes
+* Pavlik
+
+
+### Introductions
+* Emelia: I work at Inrupt with Aaron. I work on SDKs including authn SDK.
+
+
+---
+
+## Agenda
+
+### Open PRs
+
+https://github.com/solid/solid-oidc/pulls
+
+
+### Issues
+
+https://github.com/solid/solid-oidc/issues
+
+[Reconsider Client Identifiers](https://github.com/solid/solid-oidc/issues/207)
+
+
+## Minutes
+
+### Open PRs
+
+* AC: Both Pavlik and I made PRs to authentication panel to clarify UTC. I'll close mine since Pavlik's has little more details.
+
+### Reconsider Client Identifiers #207
+
+Proposal: 
+
+> The Solid-OIDC specification should drop the current definition of Client Identifiers and associated JSON-LD appendix in favor of adopting the OpenID Connect Federation definition of an Entity Identifier and Automatic Registration
+
+* AC: With OAuth2 and OIDC it is hard to globally identify clients. In most cases client_id is local to the OP or AS. In Solid we need global client identifiers, and defined them in Solid-OIDC.
+* ...: Now there is the [OpenID Federation draft](https://openid.net/specs/openid-connect-federation-1_0.html
+ ). Among other things it defines mechanism to define clients globally. The question is if we can reuse it since it. The draft is at version 24. I have specific proposal which I would like to suggest. Let's take some questions first.
+* WT: I also went through most of the spec. I want to know what people see as possible obstacles.
+* ES: With migrating to OIDC Federation, what would the migration plan look like. The server would most likely need to support current version and next. How would we deal with the federation.
+* WT: I think it shouldn't be that hard. It sould be to easy to support current and next.
+* AC: To migration path, there are 3 parts to federation when it comes to client identifiers. The client / application interacts with various endponts using client_id query parameter with URL as value. This will not change.
```

Comment:
```suggestion
* AC: As to migration path, there are 3 parts to federation when it comes to client identifiers. The client/application interacts with various endpoints using `client_id` query parameter with URL as value. This will not change.
```